new/unused_metadata.md

```diff
@@ -0,0 +1,42 @@
+---
+RFC: unassigned
+Title: Remove unused cookbook metadata
+Author: Thom May <thom@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Remove Unused Cookbook Metadata
+
+There are a set of metadata fields that have never been really supported
+by Chef, and so we should remove them.
+
+## Motivation
+
+    As a community advocate,
+    I want to ensure that cookbook maintainers focus on the things that
+    matter,
+    so that they can write better cookbooks.
+
+    As a chef developer,
+    I like removing code,
+    so that there are fewer bugs.
+
+## Specification
+
+Mark as deprecated and subsequently remove the `recommends`, `suggests`,
+`conflicts`, `replaces` and `grouping` metadata fields.
+
+## Downstream Impact
+
+Chef Server should be updated to not serve this metadata, docs need to
+be updated. Also, generators such as `chef generate`, `berks cookbook`
+need to be updated to ensure they do not create metadata files with
+those fields.
+
```

Comment:
I think Chef Server should store and send back any arbitrary key/value gunk we send it since that makes things more flexible for the future, but `knife` and `berks` should probably stop sending these.